test.py

```diff
@@ -836,10 +840,12 @@ async def run(self, options: argparse.Namespace) -> Test:
 
         self._prepare_pytest_params(options)
 
-        async with self.suite.clusters.instance() as cluster:
+        loggerPrefix = self.mode + '/' + self.uname
+        logger = LogPrefixAdapter(logging.getLogger(loggerPrefix), {'prefix': loggerPrefix})
+        async with self.suite.clusters.instance(logger) as cluster:
             try:
                 cluster.before_test(self.uname)
-                logging.info("Leasing Scylla cluster %s for test %s", cluster, self.uname)
+                logger.info("Leasing Scylla cluster %s for test %s", cluster, self.uname)
```

Comment:
I don't see how it works when we're reusing an existing cluster. The args are only passed to the build function, the adapter apparently is not reset when taking an exsiting cluster.